get-started/src/app/extensions/package.json

```diff
@@ -19,6 +19,6 @@
     "react": "^18.2.0"
   },
   "devDependencies": {
-    "@hubspot/ui-extensions-dev-server": "latest"
+    "@hubspot/ui-extensions-dev-server": "^0.0.1-beta.0"
```

Comment:
we should keep this as `latest`

Reply:
Slightly related, but you can remove the `setup` command on line 9 as it's no longer needed.